src/helper.js

```diff
@@ -22,6 +37,27 @@ ${map(formatOption, rootOptions.options).join('\n')}
 `
 }
 
+export function loadTree (tree) {
+  return mapObjIndexed(n => isModule(n) ? loadModule(n) : n)(tree)
+}
+
+export function groupTree (tree) {
+  return groupBy(([, node]) =>
+    isOptions(node) ? 'rootOptions'
+    : isCommand(node) && !isNamespace(node) ? 'root'
+    : isNamespace(node) ? 'ns'
+    : '_'
+  )(toPairs(tree))
```

Comment:
Can you expand (maybe also add comments) on the necessity of converting to pairs?